test/JDBC/expected/object_ownership.out

```diff
@@ -0,0 +1,701 @@
+-- tsql
+/** CREATE DATABASE **/
+create database object_ownership_db;
+go
+use object_ownership_db;
+go
+
+/** CREATE LOGIN, USER AND ROLE **/
+create login object_ownership_l1 with password = '123';
+go
+create user object_ownership_u1 for login object_ownership_l1;
+go
+create role object_ownership_r1;
+go
+alter role object_ownership_r1 add member object_ownership_u1;
+go
+create login object_ownership_l3 with password = '123';
+go
+create user object_ownership_u3 for login object_ownership_l3;
+go
+create login object_ownership_l4 with password = '123';
+go
+
+/** CREATE SCHEMAS WITH DIFFERENT OWNERS **/
+create schema object_ownership_s1;
+go
+create schema object_ownership_s2 authorization object_ownership_u1;
+go
+create schema object_ownership_s3 authorization object_ownership_r1;
+go
+
+/** CREATE OBJECTS IN DIFFERENT SCHEMAS USING DBO **/
+/** SCHEMA OWNER AND CURRENT USER ARE SAME **/
+create table object_ownership_s1.t(a int, b int);
+go
+create view object_ownership_s1.v as select 1;
+go
+create proc object_ownership_s1.p as select 2;
+go
+create function object_ownership_s1.f() returns int begin declare @a int; set @a = 1; return @a; END
+go
+create sequence object_ownership_s1.sq start with 1 increment by 1 ;
+go
+
+/** CREATE PARTITION FUNCTION AND SCHEME **/
+create partition function pf2 (datetime)
+as range right for values 
+(
+    '2023-01-01', 
+    '2024-01-01',
+    '2025-01-01'
+);
+go
+
+create partition scheme ps2
+as partition pf2 
+all to ([primary]);
+go
```

Comment:
Can we please add index , trigger, table type , type creation cases for objects for `/** SCHEMA OWNER AND CURRENT USER ARE SAME **/` as well?